projects/rna/in-place-connection/package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
It may be helpful to take a look at the default package.json for `wp-calypso` packages:

https://github.com/automattic/wp-calypso/blob/6850b064eaac8e5d564cb6a141b60ec158f239c6/docs/monorepo.md#L51

Namely y'all are missing `sideEffects` and a few other potentially important fields.